processor/k8sattributesprocessor/config.go

```diff
@@ -170,6 +170,17 @@ type ExtractConfig struct {
 	// OtelAnnotations extracts all pod annotations with the prefix "resource.opentelemetry.io" as resource attributes
 	// E.g. "resource.opentelemetry.io/foo" becomes "foo"
 	OtelAnnotations bool `mapstructure:"otel_annotations"`
+
+	// ServiceAttributes allows to extract service attributes based on kubernetes metadata.
+	ServiceAttributes ServiceAttributeConfig `mapstructure:"service_attributes"`
+}
+
+// ServiceAttributeConfig allows to extract service attributes based on kubernetes metadata.
+type ServiceAttributeConfig struct {
+	// Enabled enables the extraction of service attributes based on kubernetes metadata.
+	Enabled bool `mapstructure:"enabled"`
+	// WellKnownLabels enables the extraction of well-known labels as service attributes.
+	Labels bool `mapstructure:"well_known_labels"`
```

Comment:
I don't see why we need to have this configurable. One top level `ServiceAttributes` option should be enough to apply all the extraction rules, right?

Reply:
see last comment